base/io.jl

```diff
@@ -174,7 +174,12 @@ Read a single line of text, including a trailing newline character (if one is re
 the end of the input), from the given I/O stream or file (defaults to `STDIN`).
 When reading from a file, the text is assumed to be encoded in UTF-8.
 """
-readline(filename::AbstractString) = open(readline, filename)
+function readline(filename::AbstractString, chomp = false; nl2lf = false)
```

Comment:
`nl2lf` has the effect of converting all linefeeds to `\n` in output, it doesn't allow you to set which characters are considered as a newline. Adding newlines argument makes sense, but I'd like to keep to `nl2lf` or something with a better name to do normalization of line ends conveniently and efficiently.